src/Kobo/Response/MetadataResponseService.php

```diff
@@ -21,12 +28,17 @@ protected function getDownloadUrls(Book $book, KoboDevice $kobo, ?array $filters
 
         $response = [];
 
-        $formats = ['EPUB3']; // EPUB3 is required for Kobo
+        $formats = [self::EPUB3_FORMAT]; // At least EPUB3 is required for Kobo
+        if ($this->kepubifyEnabler->isEnabled()) {
```

Comment:
I am not sure if the kobo takes the EPUB3 or KEPUB  format by default. 

To be tested with a device.


(You should have advanced features such as "notes" in the kepub format)

Reply:
EPUB3 is used if present, I changed the logic to put only one format at a time.